app/views/AuthenticationWebView.tsx

```diff
@@ -173,39 +132,35 @@ class AuthenticationWebView extends React.PureComponent<IAuthenticationWebView,
 		}
 	};
 
-	render() {
-		const { loading } = this.state;
-		const { route } = this.props;
-		const { url, authType } = route.params;
-		const isIframe = authType === 'iframe';
-
-		return (
-			<>
-				<StatusBar />
-				<WebView
-					source={{ uri: url }}
-					userAgent={userAgent}
-					// https://github.com/react-native-community/react-native-webview/issues/24#issuecomment-540130141
-					onMessage={({ nativeEvent }) => this.onNavigationStateChange(nativeEvent)}
-					onNavigationStateChange={this.onNavigationStateChange}
-					injectedJavaScript={isIframe ? injectedJavaScript : undefined}
-					onLoadStart={() => {
-						this.setState({ loading: true });
-					}}
-					onLoadEnd={() => {
-						this.setState({ loading: false });
-					}}
-				/>
-				{loading ? <ActivityIndicator size='large' absolute /> : null}
-			</>
-		);
-	}
-}
+	const isIframe = authType === 'iframe';
 
-const mapStateToProps = (state: IApplicationState) => ({
-	server: state.server.server,
-	Accounts_Iframe_api_url: state.settings.Accounts_Iframe_api_url as string,
-	Accounts_Iframe_api_method: state.settings.Accounts_Iframe_api_method as string
-});
+	useLayoutEffect(() => {
+		navigation.setOptions({
+			headerLeft: () => <HeaderButton.CloseModal navigation={navigation} />,
```

Comment:
```suggestion
			headerLeft: () => <HeaderButton.CloseModal />,
```
This component does not need to pass navigation